common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -151,7 +152,7 @@ class Backend(
             }
         }
         synchronized(this@Backend) {
-            callbacks.addCallback(call, cacheKey, onSuccess to onError, randomDelay = appInBackground)
+            callbacks.addCallback(call, dispatcher, cacheKey, onSuccess to onError, randomDelay = appInBackground)
```

Comment:
this reminds me - we should probably add some jitter to the diagnostics call too. That way, if many apps get opened at the same time (say, from a remote push notification), we lessen the impact on our servers. We can probably have a more aggressive jitter too, compared to what we have for the rest of the calls.

Reply:
Good call 👍